fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -1045,7 +1045,8 @@ public void sendDropAutoIncrementMapTask() {
      * @return : table's partition name to range partition key mapping.
      */
     public Map<String, Range<PartitionKey>> getRangePartitionMap() {
-        Preconditions.checkState(partitionInfo instanceof RangePartitionInfo);
+        Preconditions.checkState(partitionInfo instanceof RangePartitionInfo, "Materialized view related base table:%s " +
```

Comment:
It is best to report errors at the upper level. Reporting MV errors here can easily cause misunderstandings.

Reply:
tks. Optimize error messages